data-solutions/data-platform-foundations/02-resources/main.tf

```diff
@@ -147,6 +166,31 @@ module "vpc-transformation" {
   subnets    = var.transformation_subnets
 }
 
+module "firewall" {
+  source               = "../../../modules/net-vpc-firewall"
+  project_id           = var.project_ids.transformation
+  network              = module.vpc-transformation.name
+  admin_ranges_enabled = false
+  admin_ranges         = [""]
+  http_source_ranges    = []
+  https_source_ranges   = []
+  ssh_source_ranges     = []
+
+  custom_rules = {
+    iap-svc = {
+      description          = "Dataflow service."
+      direction            = "INGRESS"
+      action               = "allow"
+      sources              = ["dataflow"]
+      targets              = ["dataflow"]
+      ranges               = []
```

Comment:
if this is for IAP, it should use the IAP source ranges

Reply:
Nope, this is only for dataflow